### DISCUSSION TITLE
Order by with NullsOrder.last with Managers

### DISCUSSION THREAD

**Discussion by Romain-Guillot** [simolus3/drift#3386] [Q&A] [answered]
What's the equivalent to

```dart
   final query = (select(db.myTable)
      ..orderBy([
        (t) => OrderingTerm.asc(db.myTable.rank, nulls: NullsOrder.last),
        (t) => OrderingTerm.asc(db.myTable.label),
      ])
```

with Managers ?

So, what's the best way to handle "NullsOrder.last" for this query:
```dart
    final query = db.managers.entourages
      ..orderBy(
        (o) => o.priority.asc() & o.lastName.asc() & o.firstName.asc(),
      );
```

**Comment by simolus3** [comment, marked as answer]
The manager API does not currently support the `NullsOrder` enum to control how null values are sorted. I'll add that API in https://github.com/simolus3/drift/pull/3387, so the next drift version will let you write `o.firstName.asc(nulls: NullsOrder.last)`.
